### ISSUE TITLE
Exception : Game is not installed

### ISSUE THREAD

**Issue by krio-rogue** [raid-toolkit/raid-toolkit-sdk#122]
On fresh installation raid folder name is 'raid-shadow-legends' 
Sample : `C:\_\Plarium\PlariumPlay\StandAloneApps\raid-shadow-legends\40409`
Link : `C:\_\Plarium\PlariumPlay\PlariumPlay.exe -gameid=101 -tray-start`

**Comment by krio-rogue** [comment]
Additional info:
Plarium Play v8.6.0-0.0.1 (3.201)
File `c:\_\Plarium\PlariumPlay\gamestorage.gsfn` content
```json
{
  "InstalledGames": {
    "101": {
      "Id": 101,
      "IntegrationType": 2,
      "InstallationPath": "C:\\_\\Plarium\\PlariumPlay\\StandAloneApps",
      "InsalledGames": {
        "raid-shadow-legends": "40409"
      }
    }
  },
  "Installations": []
}
```

**Comment by dnchattan** [comment]
fixed in 2.6.11 (15d64df6e1c7e61409ed9f46aec2a79a6c83de32)
